docs/DML-Operations-on-Carbon.md

```diff
@@ -104,8 +109,10 @@ Following are the options that can be used in load data:
                                  'MULTILINE'='true', 'ESCAPECHAR'='\', 
                                  'COMPLEX_DELIMITER_LEVEL_1'='$', 
                                  'COMPLEX_DELIMITER_LEVEL_2'=':',
-                                 'ALL_DICTIONARY_PATH'='/opt/alldictionary/data.dictionary'
+                                 'ALL_DICTIONARY_PATH'='/opt/alldictionary/data.dictionary',
+                                 'COLUMNDICT'='empno:/dictFilePath/empnoDict.csv, empname:/dictFilePath/empnameDict.csv'
```

Comment:
do not give this option since it can not be used together with `ALL_DICTIONARY_PATH`